spec/models/dialog_field_drop_down_list_spec.rb

```diff
@@ -134,63 +134,6 @@
     end
   end
 
-  describe "#initialize_with_values" do
```

Comment:
@eclarizio - Only thing I see, (and it's something I'm not sure I don't see)  are all of these 'red' test scenarios fully null and void?  e.g.  because of the changes in this PR we don't need them? 

Or are your new tests sufficiently covering your new version of the scenarios these tests originally covered?  

It's hard to tell from the PR as none of the test names coincide.

If I was more familiar with the process I probably wouldn't have to ask the question at all.  ☹️

Reply:
`#initialize_with_values` is no longer in use, so I've removed it and basically replaced it with `#initialize_with_given_value`, so those specs needed to be removed.